stubs/xtea/xtea/__init__.pyi

```diff
@@ -0,0 +1,25 @@
+from collections.abc import Callable
+
+from pep272_encryption import PEP272Cipher
+
+MODE_ECB: int
+MODE_CBC: int
+MODE_CFB: int
+MODE_PGP: int
+MODE_OFB: int
+MODE_CTR: int
+block_size: int
+key_size: int
+
+def new(key, **kwargs): ...
+
+class XTEACipher(PEP272Cipher):
+    block_size: int
+    IV: bytes | None
+    counter: Callable[[], bytes] | None
+    rounds: float | None
+    cycles: float | None
+    endian: str | None
```

Comment:
```suggestion
    rounds: int
    cycles: int
    endian: str
```
From the `__init__` code, looks like these can't be None and the first two are always ints.